packages/ui/src/1_atoms/Link/Link.module.css

```
@@ -0,0 +1,23 @@
+@import '../../../styles/index.css';
+
+.link {
+  @apply underline text-xs font-medium;
+}
+
+/* STYLES ================================================================== */
+
+.link.primary {
+  @apply text-primary-20;
+}
+
+.link.primary:hover {
+  @apply text-primary-10;
+}
+
+.link.secondary {
+  @apply text-secondary-20;
+}
+
+.link.secondary:hover {
+  @apply text-secondary-30;
+}
```

I think css nesting is enabled for modules and can be used like this:

```suggestion

.link {
  @apply underline text-xs font-medium;
  
  &.primary {
    @apply text-primary-20;
    &:hover {
      @apply text-primary-10;
    }
  }
  
  &.secondary {
    @apply text-secondary-20;
    &:hover {
      @apply text-secondary-30;
    }
  }
}
```